codegen/templates/api/UserInfo.ts

```diff
@@ -0,0 +1,59 @@
+import { IdentityToken } from '../models/IdentityToken'
+import { RequiredDeep } from '../models/RequiredDeep'
+import { RequestOptions } from '../models/RequestOptions'
+import http from '../utils/HttpClient'
+import OrderCloudError from '../utils/OrderCloudError'
+
+class UserInfo {
+  private impersonating: boolean = false
+
+  /**
+   * @ignore
+   * not part of public api, don't include in generated docs
+   */
+  constructor() {
+    this.GetToken = this.GetToken.bind(this)
+  }
+
+  /**
+   * Get a single address.
```

Comment:
is this address info in here by mistake?

Reply:
@erincdustin  Good catch, this is now fixed